src/main/java/de/refactoringbot/refactoring/supportedrefactorings/AddOverrideAnnotation.java

```diff
@@ -34,10 +34,11 @@ public String performRefactoring(BotIssue issue, GitConfiguration gitConfig) thr
 		FileInputStream in = new FileInputStream(gitConfig.getRepoFolder() + "/" + path);
 		CompilationUnit compilationUnit = LexicalPreservingPrinter.setup(JavaParser.parse(in));
 
-		MethodDeclaration methodDeclarationToModify = RefactoringHelper.getMethodByLineNumberOfMethodName(issue.getLine(),
-				compilationUnit);
+		MethodDeclaration methodDeclarationToModify = RefactoringHelper
+				.getMethodByLineNumberOfMethodName(issue.getLine(), compilationUnit);
 		if (methodDeclarationToModify == null) {
-			throw new BotRefactoringException("Could not find method at specified line! Automated refactoring failed.");
+			throw new BotRefactoringException(
+					"Could not find a method declaration at specified line! Please check if you placed your comment on a method declaration.");
```

Comment:
We do not know if the refactoring was triggered by a comment. It could also be the case that the analysis results are older than the code and therefore the lines do not match. That is why we must either leave out the second part of the message or make it more general.